pkg/thing/delivery/amqp/client.go

```diff
@@ -40,15 +40,20 @@ func NewMsgClientPublisher(logger logging.Logger, amqp *network.Amqp) ClientPubl
 }
 
 // SendRegisterDevice publishes the registered device's credentials to the device registration queue
+<<<<<<< HEAD
 func (mp *msgClientPublisher) SendRegisteredDevice(msg network.DeviceRegisteredResponse) error {
+=======
+func (mp *msgClientPublisher) SendRegisteredDevice(thingID, token string, errMsg *string) error {
+>>>>>>> 51d3d9f... thing: client: Fix SendRegisteredDevice signature
```

Comment:
Ooopss :grimacing: 
I think this is a leftover from `git rebase' :joy: 

```suggestion
func (mp *msgClientPublisher) SendRegisteredDevice(thingID, token string, errMsg *string) error {
```

Reply:
@lcbm I'm also forgot to rename the msg struct to `DeviceRegisteredResponse`. Now, all done! :smile: